message_definitions/v1.0/messages.xml

```diff
@@ -169,7 +169,7 @@
     </message>
 
     <message name="CAM" id="20">
-      <field name="phi" type="int16" unit="deg"/>
```

Comment:
Isn't `pan` and `tilt` a better naming rather than psi/theta/psi ? Because depending on how the you mount the camera, `phi` might be more meaningful than `psi`. But if you use pan/tilt, you can somehow assume that the required transformation of angles have been done on the autopilot side before sending the message. What do you think ?

Reply:
Yes, that sounds like a good idea. It keeps everything on the server side simple.